packages/cactus-plugin-ledger-connector-iroha2/src/test/typescript/integration/iroha2-setup-and-basic-operations.test.ts

```diff
@@ -393,4 +394,28 @@ describe("Setup and basic endpoint tests", () => {
       }),
     ).toReject();
   });
+
+  test("get prometheus exporter metrics", async () => {
+
+    const res = await env.apiClient.getPrometheusMetricsV1();
+    const promMetricsOutput =
+      "# HELP " +
+      K_CACTUS_IROHA2_TOTAL_TX_COUNT +
+      " Total transactions executed\n" +
+      "# TYPE " +
+      K_CACTUS_IROHA2_TOTAL_TX_COUNT +
+      " gauge" 
+      //+ "/n" +
```

Comment:
Why is the rest of it commented? This incomplete string cannot be parsed by the prometheus